server/gitlab/api.go

```diff
@@ -282,12 +282,58 @@ func (g *gitlab) GetProject(ctx context.Context, user *UserInfo, token *oauth2.T
 	return project, nil
 }
 
-func (g *gitlab) GetReviews(ctx context.Context, user *UserInfo, token *oauth2.Token) ([]*MergeRequest, error) {
+type LHSContent struct {
```

Comment:
See this plugin have serializers package and initialise this there. If not then the top of this file.